host/volume/api/http.go

```diff
@@ -26,6 +31,10 @@ func (api *HTTPAPI) RegisterRoutes(r *httprouter.Router) {
 	r.GET("/storage/volumes", api.List)
 	r.GET("/storage/volumes/:volume_id", api.Inspect)
 	r.PUT("/storage/volumes/:volume_id/snapshot", api.Snapshot)
+	// takes host and volID parameters, triggers a send on the remote host and give it a list of snaps already here, and pipes it into recv
+	r.POST("/storage/volumes/:volume_id/pullSnapshot", api.Pull)
```

Comment:
`pull_snapshot` is a bit better. I'm not a huge fan of the current route, but I don't have any better suggestions.